docs/guides/integration/dependency-bots.md

```diff
@@ -0,0 +1,25 @@
+# Dependency bots
+
+## Renovate
+
+uv is supported by [Renovate](https://github.com/renovatebot/renovate).
+
+With `lockFileMaintenance` enabled, Renovate will automatically detect `uv.lock` and suggest updates
+to `uv.lock` files.
+
+```json5 title="renovate.json5"
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  lockFileMaintenance: {
+    enabled: true,
+  },
+}
+```
+
+!!! note
+
+    `uv pip compile` outputs such as `requirements.txt` are not yet supported by Renovate.
```

Comment:
Created a dedicated GitHub discussion for that (https://github.com/renovatebot/renovate/discussions/30899) that will hopefully be converted to an issue, to better track the implementation. Will report back if it ends up being converted to an issue.

Reply:
An issue now exists: https://github.com/renovatebot/renovate/issues/30909.

Reply:
Awesome, I'll link to that.